comp/volunteers/refreshments.yaml

```diff
@@ -9,9 +9,17 @@ milestone: $SRYYYY Competition
 area-owner: volunteer-coordination
 
 description: >-
+  Since volunteers often don't have time to go out for lunch we should organise
+  communal food for lunch during the competition.
+
   We should have some refreshments for the volunteers at the competition.
+  Generally this should include at least tea, coffee & water as well as some biscuits & fruit.
   At a minimum this will include water and cups being available at all times
   throughout the competition including the setup day.
 
-  This should include at least tea, coffee & water as well as some biscuits &
-  fruit.
+  Required actions:
+   - [ ] Organise lunches
+   - [ ] Organise refreshments
+
+dependencies:
+  - comp/volunteers/recruit
```

Comment:
Why does the plan for refreshments depend on recruitment of volunteers? Is this to do with dietary requirements? If so, this seems like another dependency confusion resulting from the flattened structure which threatens to muddy the importance of the core of this ticket -- to ensure that volunteers who cannot leave their posts are kept at least hydrated.

Reply:
Because we need to know the number of volunteers that we are providing food and drink to.